corehq/apps/hqwebapp/crispy.py

```diff
@@ -199,8 +201,8 @@ def render(self, form, form_style, context, template_pack=None):
         for field in self.fields:
             html += render_field(field, form, form_style, context, template_pack=template_pack)
         context.update({
-            'label_html': mark_safe(self.label_html),
-            'field_html': mark_safe(html),
+            'label_html': self.label_html,
+            'field_html': mark_safe(html),  # nosec
```

Comment:
Not blocking, but how are we sure html is safe, i.e. how are we sure the output of render_field is safe? If it's guaranteed to be because of its internals, shouldn't it return a safe string? If it did return a safe string, then you could use have something like `html = format_html_join('', '{}', ((render_field(...),) for field in self.fields)` which would be more obviously safe. (Is there a more idiomatic way to pass a bunch of strings through conditional_escape, join them, and mark that safe?)